google/datalab/bigquery/_query.py

```diff
@@ -34,9 +34,8 @@ class Query(object):
   This object can be used to execute SQL queries and retrieve results.
   """
 
-  def __init__(self, sql, context=None, values=None, udfs=None, data_sources=None, **kwargs):
+  def __init__(self, sql, context=None, values={}, udfs=None, data_sources=None):
```

Comment:
> values={}
this is likely to be problematic if `values` is ever changed. please use `None` and assign an empty dictionary instead